AsyncDisplayKit/Private/_ASPendingState.mm

```diff
@@ -14,6 +14,10 @@
 #import "ASInternalHelpers.h"
 #import "ASDisplayNodeInternal.h"
 
+#define __shouldSetNeedsDisplay(layer) (flags.needsDisplay \
+  || (flags.setOpaque && opaque != (layer).opaque)\
+  || (flags.setBackgroundColor && !CGColorEqualToColor(backgroundColor, (layer).backgroundColor)))
```

Comment:
We will frequently get to the point of the equality check in evaluating this condition.  it may be worth including a pointer check; Hopefully core graphics does this internally, but it may not. This would probably catch a significant number of typical cases, because of UIColor clearColor, whiteColor, shared default values used by developers, etc.